src/librustc_codegen_llvm/type_of.rs

```diff
@@ -52,32 +52,36 @@ fn uncached_llvm_type<'a, 'tcx>(
     }
 
     let name = match layout.ty.kind {
-        ty::Closure(..) |
-        ty::Generator(..) |
-        ty::Adt(..) |
-        // FIXME(eddyb) producing readable type names for trait objects can result
-        // in problematically distinct types due to HRTB and subtyping (see #47638).
+        // FIXME(eddyb) producing readable type names for trait objects can result in
+        // problematically distinct types due to HRTB and subtyping (see #47638).
```

Comment:
Unnecessary change (could make diff smaller, up to you).

Reply:
Fixed!